packages/tc-api-graphql/index.js

```diff
@@ -7,13 +7,14 @@ const getGraphqlApi = ({
 	documentStore,
 	republishSchema,
 	republishSchemaPrefix = 'api',
+	options = {},
```

Comment:
I didn't notice something earlier. I think it makes more sense to pass them in as top level properties on the options. I can't think of any good reason why they should be wrapped in another object. Can you?

```suggestion
	typeDefs = [],
	resolvers = {},
```